src/tinystates.asm

```diff
@@ -443,3 +444,36 @@ vm:
 
 print pc, " tinysave end"
 warnpc $80FC00 ; infohud.asm
+
+print pc, " tinysave bank82 start"
+org $82FE00
+
+tinystates_preload_bg_data:
```

Comment:
print pc should be after the org

Reply:
Fixed with [28072e2](https://github.com/tewtal/sm_practice_hack/pull/94/commits/28072e250112b7de6813b160eff9a52cf79a6823)